### DISCUSSION TITLE
last nigth 26 00:22

### DISCUSSION THREAD

**Discussion by AS400JPLPC** [helix-editor/helix#11327] [Troubleshooting]
### Summary

![Capture d’écran du 2024-07-26 00-12-21](https://github.com/user-attachments/assets/b54667aa-3d00-4065-b645-92781192e766)
 up ou down scratch 

### Reproduction Steps

 

I tried this:

1. `hx`

I expected this to happen:

Instead, this happened:

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

linux

### Terminal Emulator

gnome-terminal or xfce-terminal or kitty

### Installation Method

source

### Helix Version

helix 24.7 (9b65448f)

**Comment by pascalkuthe** [comment]
if you report an issue you need to include a meaningful description and title this doesn't tell me anything

**Reply by AS400JPLPC** [reply]
thread 'main' panicked at helix-core/src/transaction.rs:483:9:
Positions [(626, Before)] are out of range for changeset len 0!
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


This is the first time I'm making a report

**Reply by pascalkuthe** [reply]
this was already reported in #11291.

Reporting a crash like this is fine but you need to create a detailed description and title. The template gives clear guidelines (explanation, reproduction steps, expected this to happen but instead that happened,...). If you open issues in the future make sure these are filled in with enough detail so that it's very clear what the problem is. 

And please stop commenting on unrelated issues. I understand you are new to opensource but that is considered bad form.

**Comment by AS400JPLPC** [comment]
I found a bug during my last nightly update. It occurs when you are
entering data and a popup is displayed. When using the "UP or DOWN" key,
there is a glitch. It works better if I wait until there are 3 characters
entered, but if the popup does not contain everything, the glitch reappears.

P.S.: I don't speak English, I use ChatGPT. I have over 45 years of
experience in computing, and I wanted to help you avoid a flood of
feedback. I use "Helix" because I find it reliable. Thank you for the great
work.

Le ven. 26 juil. 2024 à 01:01, Pascal Kuthe ***@***.***> a
écrit :

> this was already reported in #11291
> <https://github.com/helix-editor/helix/issues/11291>.
>
> Reporting a crash like this is fine but you need to create a detailed
> description and title. The template gives clear guidelines (explanation,
> reproduction steps, expected this to happen but instead that happened,...).
> If you open issues in the future make sure these are filled in with enough
> detail so that it's very clear what the problem is.
>
> And please stop commenting on unrelated issues. I understand you are new
> to opensource but that is considered bad form.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/helix-editor/helix/discussions/11327#discussioncomment-10153830>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AE7Q723RBX3N7MFG2OFZO7TZOF7THAVCNFSM6AAAAABLPOH27GVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMJVGM4DGMA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Cordialement
@bientôt JP Laroche
